### PR TITLE
Work with Python 3.5 and wxPython 4.0.1

### PR DESCRIPTION
**Pull request by zhangt58**
As of release `2.0.0`, fully support of Python 3.x and wxPython 4.x (i.e. phoenix) should be expected. I'm going to merge into `master` branch, however, all code should be compatible with Python 2.7 + wxPython classic.

### CODE REVIEW

File: beamline/ui/main.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import wx
 
 
-def run(debug=True, icon=None):
+def run(icon=None):
```
Comment (smartsammler) [inline beamline/ui/main.py:19]: .. versionchanged:: 2.0.0
    removed kwarg `debug=True`

File: beamline/models.py [hunk 2]
```diff
@@ -194,9 +194,9 @@ def flatten(ele):
         for el in ele:
             if isinstance(el, list) or isinstance(el, tuple):
                 for e in Models.flatten(el):
-                    yield(e)
+                    yield (e)
```
Comment (smartsammler) [inline beamline/models.py]: Why do you use braces after the yield?

File: beamline/ui/main.py (hunk 1 above)
Comment (zhangt58) [reply beamline/ui/main.py:19]: Good point, I've just added the note.

File: beamline/models.py (hunk 2 above)
Comment (zhangt58) [reply beamline/models.py]: I've removed the parentheses.

**Pull request merged**